README.md

```diff
@@ -102,16 +137,16 @@ XS2A Adapter is available from the Maven-Central repository. To use it in your p
     </dependencies>
 
 ```
-`service-loader` provides interfaces for communicating with banks: _AccountInformationService_ 
+`service-loader` provides interfaces for communicating with banks: _AccountInformationService_
 and _PaymentInitiationService_ for querying account data and performing payments respectively.
-`adapters` contains all implemented bank adapters and `aspsp-registry` provides the Lucene repository 
-with data records necessary for connecting with German banks. These are records for all implemented 
+`adapters` contains all implemented bank adapters and `aspsp-registry` provides the Lucene repository
+with data records necessary for connecting with German banks. These are records for all implemented
 banks at the moment.
 
-__Note__: be aware that `aspsp-registry` contains data for connecting with bank Sandboxes only, 
+__Note__: be aware that `aspsp-registry` contains data for connecting with bank Sandboxes only,
```

Comment:
I think we can omit this `Note`. It's already mentioned on line 36

Reply:
removed, thanks